test/integration/simple_operations_test.lua

```diff
@@ -490,6 +490,34 @@ pgroup.test_intermediate_nullable_fields_update = function(g)
     })
 end
 
+pgroup.test_gh_236_update_with_actual_metadata = function(g)
```

Comment:
Actually, this test doesn't verify anything. I've tried to revert the changes above and it had succeeded. I assume that Alexander example had relied on some schema migration in previous test and we need to replicate it here.

Reply:
Fixed: modified `test_intermediate_nullable_fields_update`, now this draft version not reproduce on `master`

Reply:
Well, it isn't nice. Now tests rely on the order and shuffle can change their logic. Moreover, now basic behavior tests also check non-basic case, and it won't be clear for anyone who will read this test suite without the commit history. Let's try to make a full reproducer in a separate single test, I don't think it would be too hard.

Reply:
It's okay if it would also result in subsequent cases fail before the patch

Reply:
Done